### ISSUE TITLE
Cannot start the container and successfully run hbase shell

### ISSUE THREAD

**Issue by rgcooper53** [HariSekhon/Dockerfiles#4]
Apologies if some of these issues will be obvious with more experience.  I have tried several HBase containers (including a few authored by others) and so far no luck, a variety of issues .  Note I am running RHEL7 on VMware, and I started the container this way --

docker run -t -i harisekhon/hbase-dev /bin/bash

running ./start-hbase.sh --
localhost: /hbase/bin/zookeepers.sh: line 52: ssh: command not found

ssh is indeed not in /bin, not going to make much progress like this.  Please advise, thanks.

**Comment by HariSekhon** [comment]
See the Makefile under hbase-dev which has an example usage... you don't have to specify bash or any args and it'll  automatically start HBase and open an HBase CLI Shell for you. From the hbase-dev directory you can actually just type the shorter `make run` which will start it for you as well as map all the ports so you can access them remotely.

**Comment by rgcooper53** [comment]
Thanks, I followed your directions and it is up and running now.  Should I now be able to run the hbase shell *outside* the container, i.e on my linux vm, and connect to hbase?  (Same question for my test application written in Go.)  The external shell returns an error on the list command, and the test code crashes, since clearly I'm not really connected.   I did discover that if I issue the command "curl localhost:8080" I get back the name of my test table, so I know some portion of the hbase infrastructure is visible but not all of it.  Again, your assistance is greatly appreciated, thank you.
